### DISCUSSION TITLE
Dialog with preset select after each load

### DISCUSSION THREAD

**Discussion by tzverg** [Aelto/W3_RandomEncounters_Tweaks#234] [General]
What I'm doing wrong?

**Comment by Aelto** [comment]
You're missing the mod menu, make sure to update your menu filelists. There is a section about them in the guide if you need

**Reply by tzverg** [reply]
seems, it fix my problem, thank you for help

**Reply by Aelto** [reply]
You're welcome and have fun with the mod!
